src/js/common/datetime.js

```diff
@@ -161,13 +161,11 @@ datetime = {
         var startTime = start.getTime();
         var endTime = end.getTime();
         var cursor = startTime;
-        var date = dw(startTime);
         var result = [];
 
-        while (cursor <= endTime && endTime > date.d.getTime()) {
-            result.push(new TZDate(date.d));
+        while (cursor <= endTime) {
```

Comment:
You have to revert this code.
I know your intention to fix test case failture.
But this code is necessary for summer time calculation.